demos/02-subscription-management/Controllers/NotificationsController.cs

```diff
@@ -74,7 +75,7 @@ public async Task<ActionResult<string>> Post([FromQuery] string validationToken
 
         Console.WriteLine(content);
 
-        var notifications = JsonConvert.DeserializeObject<Notifications>(content);
+        var notifications = JsonSerializer.Deserialize<Notifications>(content);
```

Comment:
The Graph SDK now has models for notifications and such. It may be worth simplifying this to remove the custom POCO.

Reply:
IDK if that's out of scope. I'll hold of merging until you confirm.

Reply:
Per your email, resolving this for now. Would be a good change for next refresh.